blockchain/blockchain-wallet-v4-frontend
# Feat/earn filter

## Description (optional)
* add earn filter to both table and history

## Testing Steps (optional)
* Go to earn tab and you should see the following:
<img width="697" alt="Screen Shot 2022-11-23 at 3 45 32 PM" src="https://user-images.githubusercontent.com/53186076/203666641-5852295e-2459-42b3-b9f1-ea08d2c2b7e6.png">

* Go to the activity page by clicking `Activity` you should see this:
<img width="1641" alt="Screen Shot 2022-11-23 at 6 10 05 PM" src="https://user-images.githubusercontent.com/53186076/203666741-a812883a-d5f9-4fd3-8bd1-222b3355e14b.png">
